src/main/java/com/tibagni/logviewer/filter/FiltersList.java

```diff
@@ -145,6 +147,7 @@ public FilterUIGroup(String groupName, Filter[] filters) {
       addBtn = new FlatButton("+");
       saveBtn = new FlatButton("Save");
       saveBtn.setVisible(false);
+      deleteBtn = new FlatButton("x");
```

Comment:
https://en.wikipedia.org/wiki/X_mark

Reply:
There are many 'x' characters in this link. I chose the one I think looks better.